go/libraries/doltcore/sqle/enginetest/stats_queries.go

```diff
@@ -299,20 +303,83 @@ var DoltHistogramTests = []queries.ScriptTest{
 	},
 }
 
-// todo: write focused tests
-// more tables, bigger tables, dolt_statitistcs table
 var DoltStatsIOTests = []queries.ScriptTest{
+	{
+		Name: "single-table",
+		SetUpScript: []string{
+			"CREATE table xy (x bigint primary key, y int, z varchar(500), key(y,z));",
+			"insert into xy values (0,0,'a'), (1,0,'a'), (2,0,'a'), (3,0,'a'), (4,1,'a'), (5,2,'a')",
+			"analyze table xy",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "select `database`, `table`, `index`, commit_hash, columns, types from dolt_statistics",
+				Expected: []sql.Row{
+					{"mydb", "xy", "PRIMARY", hash.Parse("f6la1u3ku5pucfctgrca2afq9vlr4nrs"), []string{"x"}, []string{"bigint"}},
```

Comment:
I feel like hard coding the hash into the test makes it brittle and unreadable. The test should construct the value we expect it to point to and take the hash of that value.